### PR TITLE
[2.11.x] DDF-3590 Adds suppression for CVE-2017-7525

### PR DESCRIPTION
**Pull request by vinamartin**
#### Who is reviewing it? 
@emanns95 @oconnormi 

#### Choose 2 committers to review/merge the PR. 
@bdeining
@clockard

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: dependency-check-maven-config.xml [hunk 1]
```diff
@@ -41,6 +41,17 @@
         </notes>
         <cve>CVE-2017-17485</cve>
     </suppress>
+    <suppress>
+        <notes>
+            FasterXML jackson-databind through 2.8.10 and 2.9.x through 2.9.3 allows unauthenticated remote code
```
Comment (vinamartin) [inline dependency-check-maven-config.xml]: Thoughts on better notes here?

Comment (emanns95) [reply dependency-check-maven-config.xml]: This sounds good to me

**Comment by bdeining** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by mackncheesiest** [comment]
build now (I think that last request got dropped because of the hanging build)

**Pull request merged**